src/discord/rest_api/discord_endpoints.rs

```diff
@@ -33,10 +34,11 @@ impl ApiEndpoints {
 
     pub async fn call(
         &self,
-        headers: HashMap<&str, String>,
+        headers: Option<HashMap<&str, String>>,
```

Comment:
No point in HashMap being an option and handling it, as in case the HashMap is empty request isn't going to be updated anyway